autopush/web/base.py

```diff
@@ -207,6 +210,13 @@ def _write_response(self, status_code, errno, message=None, error=None,
         if status_code == 410:
             self.set_header("Cache-Control", "max-age=86400")
 
+        if self._handling_message and status_code >= 300:
```

Comment:
This will only metric messages that are potentially stored and handled by Autopush and not passed via a bridge, correct?